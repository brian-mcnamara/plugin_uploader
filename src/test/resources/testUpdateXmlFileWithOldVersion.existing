<plugins>
    <plugin id="pluginId" url="./MyPlugin/oldFile.zip" version="0.1">
        <change-notes>someOtherNotes</change-notes>
        <description>someOtherDescription</description>
        <name>DifferentName</name>
        <idea-version since-build="0.1" until-build="0.2"/>
    </plugin>
    <plugin id="other.plugin" url="./otherPlugin/oldFile.zip" version="1.2.3">
        <name>DifferentName</name>
    </plugin>
</plugins>