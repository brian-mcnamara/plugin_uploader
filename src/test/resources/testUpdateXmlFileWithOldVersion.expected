<plugins>
    <plugin id="pluginId" url="./MyPlugin/{filename}" version="1.0">
        <change-notes>testChangeNotes</change-notes>
        <description>testDescription</description>
        <name>MyPlugin</name>
        <idea-version since-build="1.0" until-build="1.1"/>
    </plugin>
    <plugin id="other.plugin" url="./otherPlugin/oldFile.zip" version="1.2.3">
        <name>DifferentName</name>
    </plugin>
</plugins>